src/main/java/frc/robot/constants/BriefcaseConstants.java

```diff
@@ -24,6 +26,116 @@ public MotorType driveMotorType() {
         return MotorType.TALON_SRX;
     }
 
+    @Override
+    public boolean driveUseVelocity() {
+        return false;
+    }
+
+    @Override
+    public boolean driveUsePID() {
+        return false;
+    }
+
+    @Override
+    public FeedforwardConstant driveForwardRightFF() {
+        return new FeedforwardConstant(0.0, 0.0, 0.0);
```

Comment:
Do you want non-zero feedforward constants so you can watch the lights as you "drive"? Doesn't setting all zeros mean it won't drive at all?

Reply:
velocity mode is disabled on the briefcase